index.php

```diff
@@ -18,13 +18,35 @@
 $tourcms->set_base_url($BASE_URL);
 $expirationTime = time() + 600;
 
+session_start();
 
 $reserSys = new ReservationSystem($tourcms, $redis, $expirationTime);
 $templates = new Templates();
 $page = $templates->getPageUrl();
 $data = $templates->getData($page);
 
-$reserSys->checkIfChannelsExists();
+if (isset($_GET['username']) && isset($_GET['password'])) {
+    $xml = $templates->getXMLFromValidation($SCRIPT_LOGIN, $_GET);
+    if ($xml->error == 'OK') {
+        $_SESSION['username'] = $_GET['username'];
+        $_SESSION['logged'] = true;
+    }
+}
+
+if (isset($_POST['logout'])) {
+    session_start();
+    session_destroy();
+    header('Location: ' . $templates->getIndex() . '/login');
+}
+
+if ($page != 'login' && (!isset($_SESSION['logged']) || $_SESSION['logged'] != true)) {
+    error_log(print_r($_SESSION, true));
```

Comment:
This error log is not useful, please remove it

Reply:
fixed